src/main/java/org/cats/stock/controller/ReceiptController.java

```diff
@@ -19,6 +19,7 @@
  * Created by alexander on 1/4/17.
  */
 
+@CrossOrigin(origins = "*", allowedHeaders = "Origin, X-Requested-With, Content-Type, Accept")
```

Comment:
@yareda doing that on every controller is tedious. We can just create one interceptor that will filter out requests.